kn/base/templatetags/base.py

```diff
@@ -25,6 +26,15 @@ def email_filter(value):
 def mark_safe_filter(value):
     return mark_safe(value)
 
+@stringfilter
+@register.filter(name='json')
+def json_filter(data):
+    return mark_safe(json.dumps(data)
+                         .replace('&', '\u0026')
+                         .replace('<', '\u003c')
+                         .replace('>', '\u003e')
+                         .replace('/', '\u002f'))
```

Comment:
Zeg dat niet te hard: we hebben nog al wat leden die met plezier lekken zouden uitpluizen.  Zeker als we openlijk verkondigen ze te vertrouwen dit niet te doen ;).  Kun je zelf jouw huidige implementatie misbruiken?